backend/src/test/java/shook/shook/song/application/SongSearchServiceTest.java

```diff
@@ -0,0 +1,182 @@
+package shook.shook.song.application;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Stream;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.junit.jupiter.params.provider.ValueSource;
+import org.springframework.beans.factory.annotation.Autowired;
+import shook.shook.song.application.dto.SearchedSongResponse;
+import shook.shook.song.domain.Song;
+import shook.shook.song.domain.repository.SongRepository;
+import shook.shook.support.UsingJpaTest;
+
+class SongSearchServiceTest extends UsingJpaTest {
+
+    @Autowired
+    private SongRepository songRepository;
+
+    private SongSearchService songSearchService;
+
+    @BeforeEach
+    void setUp() {
+        songSearchService = new SongSearchService(songRepository);
+    }
+
+    @DisplayName("제목이 비어있을 때")
```

Comment:
💬 조금 더 자세한 설명이면 개인적으로 좋을 것 같아요!!

```suggestion
    @DisplayName("검색시 가수 이름만 존재하고 노래 제목이 없을 때")
```